FIDO2_messages_details.md

```diff
@@ -0,0 +1,188 @@
+## FIDO2 Authenticate Credential Request
+
+This message is sent as a request by the AUX MCU to authenticate a FIDO2 credential. The MAIN MCU responds
+with [FIDO2 Authenticate Credential Response](FIDO2_messages_details.md#fido2-authenticate-credential-response). The message is used to
+check whether the authenticator has the specified credential or not.
+
+**From AUX MCU:**
+
+| byte 0 - 251 | byte 252 - 283    |
+|:-------------|:------------------|
+| RPID         | Credential ID     |
+
+
+**RPID**
+
+[RPID](https://www.w3.org/TR/webauthn/#rp-id) for the request.
+
+**Credental ID**
+
+The [CredentialID](https://www.w3.org/TR/webauthn/#credential-id) for the entry to check
+
+## FIDO2 Authenticate Credential Response
+
+This message is sent as a response to request by the AUX MCU to authenticate a FIDO2 credential.
+
+**From MAIN MCU:**
+
+| byte 0 - 31    | byte 32 - 63 | byte 64 |
+|:---------------|:-------------|:--------|
+| Credential ID  | User ID      | Result  |
+
+**Credental ID**
+
+The [CredentialID](https://www.w3.org/TR/webauthn/#credential-id) for the entry to check
+
+**User ID**
+
+[User ID](https://www.w3.org/TR/webauthn/#dom-publickeycredentialuserentity-id) associated witht the credential
+
+**Result**
+
+| Value  | Description                                |
+|:-------|:-------------------------------------------|
+| 0x00   | Authenticator does not have the credential |
+| 0x01   | Authenticator does have the credential     |
+
+## FIDO2 Make Authenticator Data Request
+
+This message is sent as a request by the AUX MCU to create authenticator data for a FIDO2 credential. The MAIN MCU
+responds with [FIDO2 Make Authenicator Data Response](FIDO2_messages_details.md#fido2-make-authenticator-data-response). See FIDO2 specification
+for definition of the values that are not explicitly defined here.
+
+**From AUX MCU:**  
+
+| byte 0 - 251 | byte 252 - 316    | byte 317 - 381 | byte 382 - 413        | byte 414 - 446   | byte 447 | byte 448 |
+|:-------------|:------------------|:---------------|:----------------------|------------------|:---------|:---------|
+| RPID         | User ID           | User name      | Display name          | Client data hash | Index    | New cred |
+
+**RPID**
+
+[RPID](https://www.w3.org/TR/webauthn/#rp-id) for the request.
+
+**User ID**
+
+[User ID](https://www.w3.org/TR/webauthn/#dom-publickeycredentialuserentity-id) associated witht the credential
+
```

Comment:
User name description missing

Reply:
I just provide the link...no need to repeat? :-)

Reply:
for the user name? didn't see it :(